### ISSUE TITLE
Show hardware or usage of selected node on server creation page.

### ISSUE THREAD

**Issue by BolverBlitz** [Ctrlpanel-gg/panel#289]
# 🚀 Feature Request

### Is your proposal related to a problem?
No

*Provide a clear and concise description of what the problem is.
When a node is selected, there could be a smal summery of CPU Type, Ram Type, Disk Type and usage of that node.
Also maybe show the location of that node infront.

*Provide a clear and concise description of what you want to happen.
As shown in the picture, something like so. The usage would be enoth to either show the "At-A-Glance" Section of ptero*
or it could show relation of total servers and servers running on that node. 
The usage would just be great to give the user a tiny bit of information of what node to pick.
![grafik](https://user-images.githubusercontent.com/35345288/141596006-e5c4c66a-cede-46d6-851c-f225af199d3c.png)

At-A-Glance of Ptero: 
![grafik](https://user-images.githubusercontent.com/35345288/141595460-4dae2a74-8979-4598-8966-0e282ae044ef.png)

**Comment by BolverBlitz** [comment]
Yes, i´m aware of the fact that the hardware part could be added in pterodactyl as name of that node but i personaly disslike that idear.

**Comment by anondev-sudo** [comment]
Usage in % of the node eould be good. When the server will exceed allocated ram and storage limits, show error that it would exceed node allocation limit, and tell user to pick different node.

As now you can overallocate nodes, and go over the limit you set on ptero.

**Comment by 1day2die** [comment]
im pretty sure pterodactyl offers no such API?

**Comment by BolverBlitz** [comment]
They do. 
https://panel.ebg.pw/api/application/nodes/5

Nodes Avaibke Resources:
```json
"memory": 131072,
"memory_overallocate": 250,
"disk": 819200,
 "disk_overallocate": 5,
```

Currently allocated Resources:
```json
"allocated_resources": {
"memory": 12288,
"disk": 40960
}
```

**Comment by S0ly** [comment]
this is pretty important imo

**Comment by LakshmiBhaskarPVL** [comment]
i can implement this

**Comment by S0ly** [comment]
it cannot be done I think whit the new region system

**Comment by S0ly** [comment]
also now I dont see why it would be useful
